etc/profile-a-l/keepassxc.profile

```diff
@@ -93,6 +93,8 @@ private-etc
 private-tmp
 
 dbus-user filter
+dbus-user.own org.freedesktop.Secret.*
```

Comment:
Is this path actually exposed/needed? All our other profiles that support this API only use `dbus-user.talk org.freedesktop.secrets` (cfr. [this](https://specifications.freedesktop.org/secret-service/latest/ch12.html)). Note the difference with `dbus-user.own`...

Reply:
You're right! I've tested it without this line and it still works as intended. I've removed it.